private/objectiveWrap.m

```diff
@@ -18,6 +18,7 @@
     objSign = wrapperOptions{1};
     fixedTheta = wrapperOptions{2};
     outNumber = wrapperOptions{4};
+    optimizer = wrapperOptions{5};
     countErrors = wrapperOptions{6};
     showWarnings = wrapperOptions{7};
     if isempty(fixedTheta)
```

Comment:
correct me if I am wrong but I thought that everything related to options.fixedParameters is taken care of in getReduced...

Reply:
Not yet:
Jan made up the corresponding files in issue #121 , but so far these functions are not yet used in Pesto. I suggest leaving them out in this PR and addressing issue #121 in another PR.

Reply:
I agree, this is beyond scope of this pull request